test/unit/webdriver/device/keyboard_test.py

```diff
@@ -32,3 +32,65 @@ def test_hide_keyboard(self):
             appium_command('/session/1234567890/appium/device/hide_keyboard')
         )
         assert isinstance(driver.hide_keyboard(), WebDriver)
+
+    @httpretty.activate
+    def test_press_keycode(self):
+        driver = android_w3c_driver()
+        httpretty.register_uri(
+            httpretty.POST,
+            appium_command('/session/1234567890/appium/device/press_keycode'),
+            body='{keycode: 86}'
```

Comment:
Here can be `json.dumps(dict(keycode=86))` to buld JSON or write JSON like `'{"keycode": "86"}'`

Reply:
do you think the below works?
`    def test_press_keycode(self):
        driver = android_w3c_driver()
        httpretty.register_uri(
            httpretty.POST,
            appium_command('/session/1234567890/appium/device/press_keycode'),
            body='{"value": "86"}'
        )
        driver.press_keycode('86')
        d = get_httpretty_request_body((httpretty.last_request()))
        assert d['keycode'] == 86`

Reply:
The above test passes

Reply:
yes 🆗

Reply:
@KazuCocoa Done.